package.json

```diff
@@ -8,7 +8,8 @@
     "start": "next start",
     "start:production": "USE_CHROMIUM_PATH=true next start",
     "lint": "next lint",
-    "deploy": "docker build"
+    "deploy": "docker build",
+    "oxlint": "oxlint"
```

Comment:
I would recommend against using any linter which is not ESLint for projects at this time due to how developers and teams have existing knowledge which can't be built on here. It's barely used at this point and I don't see how this projects benefits from the linter being 1000x faster.

https://npmtrends.com/eslint-vs-oxlint

Reply:
I think it's worth considering oxlint because it's a single dependency which avoids annoying (peer-) dependency conflicts when upgrading, and oxlint is basically just an ESLint reimplementation so anyone experienced with ESLint will be quickly familiar with oxlint. My experience with oxlint has been better than ESLint (so far), mostly because of the faster feedback.

I don't feel *too* strongly about this though. Open to hear other opinions.